RNAseqWorkflow.iml

```diff
@@ -2,7 +2,7 @@
 <module type="JAVA_MODULE" version="4">
```

Comment:
This is only for IntelliJ. Please ignore. By convention we have these configuration files, because we all used to use IntelliJ IDEA as our IDE.